ecosystem/sep-0010.md

```diff
@@ -199,6 +204,8 @@ Upon successful verification, **Server** responds with a session JWT, containing
 * `iat` (the time at which the JWT was issued [RFC7519, Section 4.1.6](https://tools.ietf.org/html/rfc7519#section-4.1.6)) — current timestamp (`1530644093`)
 * `exp` (the expiration time on or after which the JWT must not be accepted for processing, [RFC7519, Section 4.1.4](https://tools.ietf.org/html/rfc7519#section-4.1.4)) — a server can pick its own expiration period for the token (`1530730493`)
 * `client_domain` - (optional) a nonstandard JWT claim containing the client home domain, included if the challenge transaction contained a `client_domain` (see [Verifying Client Application Identity](#verifying-client-application-identity))
+* `memo_type` — (optional) the memo type provided in the challenge, one of `text`, `id` or `hash`. See the [Memos](#memos) section for more information.
+* `memo` — (optional) the memo provided in the challenge, always of type `string` (even when `memo_type` is equal to `id`) to support parsing the value in languages that don't support big numbers.
```

Comment:
💡 I would make the why a little clearer here, the issue is actually that the format of JWTs, JSON, doesn't support 64-bit integers, not just languages. But what you have is accurate, so I don't feel strongly about this.
```suggestion
* `memo` — (optional) the memo provided in the challenge, always of type `string` even when `memo_type` is equal to `id` (unsigned 64-bit integer) because JSON and some runtimes do not support 64-bit integers.
```

Reply:
> > The claims in a JWT are normally statements about the subject.
> 
> You also have to squint hard to think that the `memo` and `memo_type` claims are _about_ the shared account holder, since they're really changing the identity of the subject

Claims should be attributes of the subject that have been proven. In the context of SEP-10 the memo isn't an attribute about the account, it's a modifier that changes the scope of authorization. This compels me to think embedding within the sub is more important than I had first thought.

Reply:
I agree. We'll update the format 👍